foodplanner-api/Controller/UsersController.cs

```diff
@@ -0,0 +1,61 @@
+using foodplanner_api.Models;
+using foodplanner_api.Service;
+using Microsoft.AspNetCore.Mvc;
+
+namespace foodplanner_api.Controller;
```

Comment:
Nitpick: Kan vi omdøbe til FoodplannerAPI for at følge dotnet standard? Hvis ikke der er mulighed for et global rename, så bare ignorér.
```suggestion
namespace FoodplannerAPI.Controller;
```